assets/javascripts/discourse/templates/components/chat-live-pane.hbs

```diff
@@ -80,21 +80,43 @@
 
 {{#if expanded}}
   {{#if selectingMessages}}
-    <div class="chat-selection-management-buttons">
-      {{d-button
-        id="chat-move-to-topic-btn"
-        class="btn-primary"
-        label="chat.selection.move_to_topic"
-        title="chat.selection.move_to_topic"
-        disabled=moveToTopicDisabled
-        action=(action "moveMessagesToTopic")
-      }}
-      {{d-button
-        class="btn-secondary cancel-btn"
-        label="chat.selection.cancel"
-        title="chat.selection.cancel"
-        action=(action "cancelSelecting")
-      }}
+    <div class="chat-selection-management">
+      <div class="chat-selection-management-buttons">
+        {{#if canMoveToTopic}}
+          {{d-button
+            id="chat-move-to-topic-btn"
+            class="btn-primary"
+            icon="sign-out-alt"
+            label="chat.selection.move_to_topic"
+            title="chat.selection.move_to_topic"
+            disabled=(not anyMessagesSelected)
+            action=(action "moveMessagesToTopic")
+          }}
+        {{/if}}
+        {{#if canQuote}}
+          {{d-button
+            id="chat-quote-btn"
+            class="btn-secondary"
+            icon="quote-left"
+            label="chat.selection.quote_selection"
+            title="chat.selection.quote_selection"
+            disabled=(not anyMessagesSelected)
+            action=(action "quoteMessages")
+          }}
+        {{/if}}
+        {{d-button
+          icon="times"
+          class="btn-secondary cancel-btn"
+          label="chat.selection.cancel"
+          title="chat.selection.cancel"
+          action=(action "cancelSelecting")
+        }}
+      </div>
+      {{#if showChatQuoteSuccess}}
+        <div class="chat-selection-message alert alert-success">
+          {{i18n "chat.quote.copy_success"}}
+        </div>
+      {{/if}}
     </div>
```

Comment:
I'm not sure how hard it is to extract values out, and I understand this PR is already quite big, but this div seems like it could be a component on its own? If it can be done it'll be nice so the `chat-live-pane` doesn't get too gnarly when it grows even bigger.

Reply:
Yeah that makes sense, should probably be called `chat-selection-management` component